homotopy-web/src/app/signature/item.rs

```diff
@@ -283,8 +311,8 @@ impl ItemView {
                 <input
                     type="text"
                     class="signature__item-name-input"
-                    value={name.clone()}
-                    oninput={ctx.link().callback(move |e: InputEvent| {
+                    value={self.name.clone()}
```

Comment:
`value={&self.name}` might work here too?

Reply:
Yew doesn't like this for whatever reason. Annoying.

Reply:
Seems like Yew wants `&'static` references only for props.